ote_sdk/ote_sdk/utils/dataset_utils.py

```diff
@@ -32,8 +32,16 @@ def split_local_global_dataset(dataset) -> Tuple[DatasetEntity, DatasetEntity]:
     for gt_item in dataset:
 
         annotations = gt_item.get_annotations()
-        global_annotations = [annotation for annotation in annotations if Rectangle.is_full_box(annotation.shape)]
-        local_annotations = [annotation for annotation in annotations if not Rectangle.is_full_box(annotation.shape)]
+        global_annotations = [
```

Comment:
When you do this split, do you change somewhat the original content of the resultset.ground_truth_dataset and resultset.prediction_dataset?

Reply:
Thanks Dick. Quite clear. And also, the AnomalyLocalizationPerformance will then be assigned to the original resultset, correct?

Reply:
correct